xbmc/dbwrappers/dataset.cpp

```diff
@@ -321,25 +335,64 @@ bool Dataset::set_field_value(const char *f_name, const field_value &value) {
   //  return false;
 }
 
+/********* GUESS SECTION START *********/
+bool Dataset::get_guess(const char *f_name) {
+  if(guess_enabled && !(ds_state == dsEdit || ds_state == dsInsert))
```

Comment:
spaces here and there, please use our formatting conventions http://kodi.wiki/view/Code_guide-lines_and_formatting_conventions#Whitespaces